### PR TITLE
Added Geolocation of two Mentors

### PR DESCRIPTION
**Pull request by NaviRocker**
Preview Link - https://naveentherocker.github.io/gci17.fossasia.org/

### CODE REVIEW

**Review by anshumanv — APPROVED** [review]

**Comment by NaviRocker** [comment]
@anshumanv How about merging it?

**Comment by anshumanv** [comment]
Let's gather one more approval @naveentherocker

**Review by rupav — APPROVED** [review]
Great Work 😃.

**Comment by anshumanv** [comment]
Just resolve conflicts and this is good to go.

File: _data/students.yml [hunk 1]
```diff
@@ -145,7 +145,16 @@
   country: Singapore
   github: ChenNupde
   image: Nuode.jpg
-  
+
+  name: Naveen Rajan
```
Comment (anshumanv) [inline _data/students.yml:146]: You are already in students?

Comment (NaviRocker) [reply _data/students.yml:146]: Well I just added My Self

Comment (anshumanv) [reply _data/students.yml:146]: https://github.com/naveentherocker/gci17.fossasia.org/blob/43c31a59e61713a2d17274c468360863d672c802/_data/students.yml#L27

Comment (anshumanv) [reply _data/students.yml:146]: In your fork, make updates here.

Comment (NaviRocker) [reply _data/students.yml:146]: Now it's fine

**Review by anshumanv — APPROVED** [review]

**Review by the-codingninja — APPROVED** [review]
LGTM!!

**Comment by the-codingninja** [comment]
Please try to squash commits as you are familiar with PR's you should try to follow best practices

**Pull request merged**